File: codewars/Find Array.js
function findArray(arr1, arr2) {
  let newArray = [];
  for (let x of arr2) {
    if (x < arr1.length) {
      newArray.push(arr1[x]);
    }
  }
  return newArray;
}

console.log(findArray([1, 2, 3, 4, 5], [1, 2, 3, 4, 5, 6, 7, 8, 9]));

File: codewars/Find Array.py
def find_array(arr1, arr2):
    newArray = []

    for x in arr2:
        if x < len(arr1):
            newArray.append(arr1[x])
    return newArray


# one liner
def find_array(arr1, arr2):
    return [arr1[x] for x in arr2 if x < len(arr1)]